File: w06/d03/Instructors/fizz_buzz.js
var n = 0;

var say = function(str) {
  console.log(str);
};

while (n <= 100) {

  if (n % 3 == 0 && n % 5 == 0) {
    console.log("FizzBuzz");
  } else if (n % 3 == 0) {
    console.log("Fizz");
  } else if (n % 5 == 0) {
    console.log("Buzz"); 
  } else {
    console.log(n);
  };

  n = n + 1;
};

File: w06/d03/Instructors/fizz_buzz.rb
n = 0

def say(str)
  puts str
end

while n <= 100

  if n % 3 == 0 && n % 5 == 0
    puts "FizzBuzz"
  elsif n % 3 == 0
    puts "Fizz"
  elsif n % 5 == 0
    puts "Buzz"
  else
    puts n
  end

  n = n + 1
end